app/code/Magento/LoginAsCustomer/Test/Mftf/Test/StorefrontStickyLoginAsCustomerNotificationBannerTest.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="StorefrontStickyLoginAsCustomerNotificationBannerTest">
+        <annotations>
+            <features value="Login as Customer"/>
+            <useCaseId value="https://github.com/magento/magento2/issues/29354"/>
+            <stories value="Availability of sticky UI elements if module enable/disable"/>
+            <title value="Sticky Notification Banner is present on Storefront page"/>
+            <description
+                value="Verify that Sticky Notification Banner is present on page if 'Login as customer' functionality used"/>
+            <testCaseId value=""/>
+            <severity value="CRITICAL"/>
+        </annotations>
+        <before>
+            <magentoCLI command="config:set {{LoginAsCustomerConfigDataEnabled.path}} 1"
+                        stepKey="enableLoginAsCustomer"/>
+            <actionGroup ref="CliCacheFlushActionGroup" stepKey="flushConfigCache">
+                <argument name="tags" value="config"/>
+            </actionGroup>
+            <createData entity="Simple_US_Customer" stepKey="createCustomer"/>
+            <actionGroup ref="AdminLoginActionGroup" stepKey="login"/>
+        </before>
+
+        <after>
+            <deleteData createDataKey="createCustomer" stepKey="deleteCustomer"/>
+            <magentoCLI command="config:set {{LoginAsCustomerConfigDataEnabled.path}} 0"
+                        stepKey="disableLoginAsCustomer"/>
+            <actionGroup ref="CliCacheFlushActionGroup" stepKey="flushConfigCache">
```

Comment:
I would strongly recommend [cleaning](https://github.com/magento/magento2/blob/e9190a5bb2c4aede2a752e94a3e87dcf9647b604/app/code/Magento/Backend/Test/Mftf/ActionGroup/CliCacheCleanActionGroup.xml#L11) cache by tag instead of flushing it. The invalidation process is always much faster than wiping out.